packages/front-end/components/Queries/ExpandableQuery.tsx

```diff
@@ -93,17 +103,37 @@ const ExpandableQuery: FC<{
             Took{" "}
             {formatDistanceStrict(
               getValidDate(query.startedAt),
-              // @ts-expect-error TS(2345) If you come across this, please fix it!: Argument of type 'Date | undefined' is not assigna... Remove this comment to see the full error message
               getValidDate(query.finishedAt)
             )}
           </em>
         </small>
       )}
       {query.status === "running" && (
-        <div className="alert alert-info">
+        <>
+          <div className="alert alert-info">
+            <em>
+              Running for{" "}
+              {formatDistanceStrict(getValidDate(query.startedAt), new Date())}
+            </em>
+            {query.dependencies?.length ? (
+              <div>
+                <em>
+                  Was queued for{" "}
+                  {formatDistanceStrict(
+                    getValidDate(query.createdAt),
+                    getValidDate(query.startedAt)
+                  )}
```

Comment:
This will kind of break for cached copies since `createdAt` will be after `startedAt`

Reply:
yeah, but it'll almost never be in "running" state? But I'm also happy to just hide this if the query has a cached query that was used (I made this change).